src/schema/posts.ts

```diff
@@ -797,7 +797,7 @@ export const typeDefs = /* GraphQL */ `
       title: String!
 
       """
-      Content of the post (max 4000 chars)
+      Content of the post (max 10000 chars)
```

Comment:
I think we can remove (max 10000 chars) part, does not really add to anything, just internal label
```suggestion
      Content of the post
```